src/slot_checker.py

```diff
@@ -72,6 +77,7 @@ def wrapper(*args, **kwargs):
     @check_signin
     def get_project_slots(self, project, start, end):
         if project == 'debug_my_slots':
+        if DEBUG_LEVEL == log.DEBUG:
```

Comment:
Maybe a merge error:

File "/usr/src/app/./slot_checker.py", line 80
if DEBUG_LEVEL == log.DEBUG:
IndentationError: expected an indented block

The correct line is:
if project == 'debug_my_slots':

Reply:
definitely a rebase error